content/news/playtest-20190106.md

```diff
@@ -0,0 +1,46 @@
+Better late than never, OpenRA is celebrating the new year with our planned "2018 holiday-season" playtest. [Playtest 20190106](/download/) brings some big changes to gameplay mechanics, more single-player content, and even more bug fixes and performance improvements.
+
+The biggest change comes to the unit targeting logic, which has been reworked to better understand the Fog of War. This has fixed a collection of long-standing issues (especially with artillery-style units), but we expect will impact the overall game balance in subtle ways that we can't predict. We hope that the community can rally behind testing and report any balance issues so that they can be addressed before the final release.
```

Comment:
Just a suggestion with a little different wording and more focus on explaining.

`This has fixed a collection of long-standing issues (especially with artillery-style units and cloaked structures) that have been blocking long-planned engine advancements. However, due to the subtle nature of these changes, more testing is required to predict the impact on overall gameplay and game balance . We hope that the community can rally behind testing and report any issues so that they can be addressed before the final release.`